### PR TITLE
Windup 2961 default tls

### PR DESCRIPTION
**Pull request by jonathanvila**
Default TLS configuration for ingresses ( available on OCP 4.6 + )
https://issues.redhat.com/browse/WINDUP-2961

### CODE REVIEW

**Comment by m-brophy** [comment]
secure-mta Ingress created with yaml as specified in the JIRA

File: src/main/resources/k8s/def/windup.deployment.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ spec:
     spec:
       serviceAccountName: windup-operator
       containers:
-        - image: quay.io/windupeng/windup-operator-native:latest
+        - image: quay.io/windupeng/windup-operator-native:0.0.2
```
Comment (m-brophy) [inline src/main/resources/k8s/def/windup.deployment.yaml:20]: Do we want to deploy this particular version?

Comment (jonathanvila) [reply src/main/resources/k8s/def/windup.deployment.yaml:20]: Good catch.
This file is only intended for manual deployment of the operator. So I thought it was wrong to have latest and we should have 0.0.2 or stable. 
We are going to get rid of the "stable" tag, but for the moment I will modify it as "stable" as it's the one really deployed by operatorhub.
wdyt ?

Comment (m-brophy) [reply src/main/resources/k8s/def/windup.deployment.yaml]: Great, thanks.

**Review by m-brophy — APPROVED** [review]

**Pull request merged**